rust_dev_preview/examples/sending-presigned-requests/src/main.rs

```diff
@@ -150,8 +151,20 @@ async fn send_presigned_request_with_reqwest(
 ) {
     let client = reqwest::Client::new();
     let res = client
```

Comment:
I'd probably go to http_02x then use `.try_into()` to get a reqwest::Reqwest. You can send it with the execute API.

Reply:
When PresignedRequest impls Clone we can do that for this example.